cicd/3-app/javabuilder/template.yml.erb

```diff
@@ -857,18 +858,19 @@ Resources:
     Properties:
       AlarmName: !Sub "${SubdomainName}_<%=name.downcase%>_high_concurrent_executions"
       AlarmDescription: !Sub |
-          Alarm if javabuilder usage exceeds 400 concurrent
-          executions for 10 minutes. Occasional spikes are expected, but
-          long-running high usage is an indication of an attack. Page the student learning
-          team for further investigation. See this doc for investigation steps
+          Alarm if javabuilder usage has high concurrent executions for 10 minutes.
+          Occasional spikes are expected, but long-running high usage is an indication
+          of an attack. If this is occuring on the demo environment, this is a non-urgent
+          issue as we expect occasional periods of high usage. If it is on production, 
+          page the student learning team for further investigation. See this doc for investigation steps
           https://docs.google.com/document/d/1bHvV6pvUcwxgZpw0YWBmxFggQL5KqYx9zwolwkZhjU8/edit#bookmark=id.xs1gcuxrw6ze
       ActionsEnabled: true
       AlarmActions:
-          - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-Urgent"]
+          - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !If [IsDemoCondition, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:javabuilder-demo-high-concurrency", !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-Urgent"]]
       EvaluationPeriods: 10
       DatapointsToAlarm: 10
       Period: 60
-      Threshold: 400
+      Threshold: !If [IsDemoCondition, 45, 400]
```

Comment:
To eliminate `IsDemoCondition`:

Parameterize `HighConcurrentExecutionsAlarmThreshold`.